setup.py

```diff
@@ -2,23 +2,54 @@
 from setuptools import setup
 from setuptools import find_packages
 
-README = open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
-REQUIREMENTS = [line.strip() for line in
-                open("requirements.txt").readlines()]
-
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
+def is_requirement(line):
+    """
+    Return True if the requirement line is a package requirement;
+    that is, it is not blank, a comment, or editable.
+    """
+    # Remove whitespace at the start/end of the line
+    line = line.strip()
+
+    # Skip blank lines, comments, and editable installs
+    return not (
+        line == '' or
+        line.startswith('-r') or
+        line.startswith('#') or
+        line.startswith('-e') or
+        line.startswith('git+')
+    )
+
+def load_requirements(*requirements_paths):
+    """
+    Load all requirements from the specified requirements files.
+    Returns a list of requirement strings.
+    """
+    requirements = set()
+    for path in requirements_paths:
+        requirements.update(
+            line.strip() for line in open(path).readlines()
+            if is_requirement(line)
+        )
+    return list(requirements)
+
+README = open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
+REQUIREMENTS = load_requirements('requirements.txt').append('Django>=1.8,<2')
```

Comment:
I added the Django requirement here explicitly, rather than in tox-requirements.txt.